src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py

```diff
@@ -710,3 +710,343 @@ def test_backup_disk_exclusion(self, resource_group, vault_name, vm_name, storag
+
+        self.assertIn(self.kwargs['identity1_id'], userMSI)
+        self.assertIn(self.kwargs['identity2_id'], userMSI)
+        self.assertIn(self.kwargs['identity3_id'], userMSI)
+
+        userMSI_json = self.cmd('backup vault update --remove-user-assigned --identity-id {identity2_id} {identity3_id} -g {rg} -v {vault}', checks=[
+            self.check("identity.type", "SystemAssigned, UserAssigned"),
+            self.check("properties.provisioningState", "Succeeded")
+        ]).get_output_in_json()
+
+        userMSI = list(userMSI_json['identity']['userAssignedIdentities'].keys())
+
+        self.assertIn(self.kwargs['identity1_id'], userMSI)
+
+        self.cmd('backup vault update --remove-user-assigned --identity-id {identity1_id} -g {rg} -v {vault}', checks=[
+            self.check("identity.type", "SystemAssigned"),
+            self.check("properties.provisioningState", "Succeeded")
+        ])
+
+        self.cmd('backup vault update --remove-system-assigned -g {rg} -v {vault}', checks=[
+            self.check("identity.type", "None"),
+            self.check("properties.provisioningState", "Succeeded")
+        ])
+
+        self.cmd('backup vault update --identity-type systemassigned -g {rg} -v {vault}', checks=[
+            self.check("identity.type", "SystemAssigned"),
+            self.check("properties.provisioningState", "Succeeded")
+        ])
+
+        self.cmd('backup vault update --identity-type None -g {rg} -v {vault}', checks=[
+            self.check("identity.type", "None"),
+            self.check("properties.provisioningState", "Succeeded")
+        ])
+
+
+    @ResourceGroupPreparer()
+    @VaultPreparer(parameter_name='vault1')
+    @VaultPreparer(parameter_name='vault2')
+    def test_backup_encryption(self, resource_group, resource_group_location, vault1, vault2):
+        self.kwargs.update({
+            'loc' : resource_group_location,
+            'vault1': vault1,
+            'vault2': vault2,
+            'rg': resource_group,
+        })
+
+
+        self.kwargs['identity1'] = self.create_random_name('clitest-identity', 50)
+        self.kwargs['identity1_id'] = self.cmd('identity create -n {identity1} -g {rg} --query id').get_output_in_json()
+        self.kwargs['identity1_principalid'] = self.cmd('az identity show -n {identity1} -g {rg} --query principalId').get_output_in_json()
+
+        self.kwargs['identity2'] = self.create_random_name('clitest-identity', 50)
+        self.kwargs['identity2_id'] = self.cmd('identity create -n {identity2} -g {rg} --query id').get_output_in_json()
+        self.kwargs['identity2_principalid'] = self.cmd('az identity show -n {identity2} -g {rg} --query principalId').get_output_in_json()
+
+
+        userMSI_v1_json = self.cmd('backup vault update --identity-type userassigned --identity-id {identity1_id} {identity2_id} -g {rg} -v {vault1}', checks=[
+            self.check("identity.type", "UserAssigned"),
```

Comment:
not needed as we are already doing such validations in test_backup_identity

Reply:
So I should just update the vault and remove the checks part?